packages/@okta/vuepress-site/docs/guides/common-hook-set-up-steps/main/index.md

```diff
@@ -71,10 +71,18 @@ The inline hook guides use [HTTP Basic Authentication](/books/api-security/authn
 
 For example, the credential pair used in the inline hook examples is `admin:supersecret`, which encoded in Base64 is `YWRtaW46c3VwZXJzZWNyZXQ=`. Adding the scheme to this value creates the inline hook **Authentication secret** value: `Basic YWRtaW46c3VwZXJzZWNyZXQ=`.
 
-After including the `npm` packages, add the following code snippet in your project.
+To add HTTP Basic Authentication to your external service:
+
+1. Include the `npm` package `express-basic-auth` to the `package.json` file in the left-hand project menu.
```

Comment:
do we tell them which version or is that user-specific? "express-basic-auth": "^1.2.0"

```suggestion
1. Include the `npm` package `express-basic-auth` in the `package.json` file in the left-hand project menu.
```